x-pack/plugins/fleet/server/services/agent_policy.ts

```diff
@@ -183,7 +183,11 @@ class AgentPolicyService {
     if (options.bumpRevision || options.removeProtection) {
       await this.triggerAgentPolicyUpdatedEvent(soClient, esClient, 'updated', id);
     }
-    logger.debug(`Agent policy ${id} update completed`);
+    logger.info(
+      `Agent policy ${id} update completed, revision: ${
+        options.bumpRevision ? existingAgentPolicy.revision + 1 : existingAgentPolicy.revision
+      }`
+    );
```

Comment:
I think moving this to `info` makes sense, but does this result in any massive log spam during bulk updates or anything like that?

Reply:
It is one log per policy per revision, so shouldn't add too many logs. I changed these to info mostly for testing, I can turn it down a little before merging.
<img width="1247" alt="image" src="https://github.com/elastic/kibana/assets/90178898/af51ed69-60aa-449b-b5eb-ce19929d68e4">